Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2.yml

```diff
@@ -1011,7 +1011,312 @@ script:
+                    eDiscovery: eDiscovery (also called eDiscovery Standard) cases are used to manage legal or other types of investigations.
+                    This is the default value.
+                    InsiderRisk: Insider risk cases are used to manage insider risk management cases.
+                    Typically, insider risk management cases are manually created in the Microsoft Purview
+                    compliance portal to further investigate activity based on a risk alert.
+                    SupervisionPolicy: This type of case corresponds to communication compliance policy."
+      required: false
+      defaultValue: "eDiscovery"
+      isArray: false
+      auto: PREDEFINED
+      predefined:
+      - AdvancedEdiscovery
+      - ComplianceClassifier
+      - DataInvestigation
+      - DSR
+      - eDiscovery
+      - InsiderRisk
+      - SupervisionPolicy
+    - name: description
+      description: Case description.
+      required: false
+      isArray: false
+    - name: external_id
+      description: Case external Id.
+      required: false
+      isArray: false
+    outputs:
+    - contextPath: O365.SecurityAndCompliance.ComplianceCase.Name
+      description: Case name.
+      type: String
+    - contextPath: O365.SecurityAndCompliance.ComplianceCase.Status
+      description: Case status.
+      type: String
+    - contextPath: O365.SecurityAndCompliance.ComplianceCase.CreatedDateTime
+      description: Case created date time.
+      type: String
+  - name: o365-sc-compliance-case-list
+    deprecated: false
+    description: List different types of compliance cases in the Microsoft Purview compliance portal 
+    execution: false
+    arguments:
+    - name: identity
+      description: List cases by identity.
+      required: false
+      isArray: false  
+    - name: case_type
+      description: List cases by type.
+      required: false
+      isArray: true
+      auto: PREDEFINED
+      predefined:
+      - AdvancedEdiscovery
+      - ComplianceClassifier
+      - DataInvestigation
+      - DSR
+      - eDiscovery
+      - InsiderRisk
+      - SupervisionPolicy
+    - name: limit
+      description: Limit returned cases list size.
+      required: false
+      isArray: false
+      defaultValue: 50
+    outputs:
+    - contextPath: O365.SecurityAndCompliance.ComplianceCase.Name
+      description: Case name.
+      type: String
+    - contextPath: O365.SecurityAndCompliance.ComplianceCase.Status
+      description: Case status.
+      type: String
+    - contextPath: O365.SecurityAndCompliance.ComplianceCase.GUID
+      description: Case GUID.
+      type: UUID
+    - contextPath: O365.SecurityAndCompliance.ComplianceCase.CreatedDateTime
+      description: Case created date time.
+      type: String
+  - name: o365-sc-compliance-case-delete
+    deprecated: false
+    description: Removes compliance cases from the Microsoft Purview compliance portal or the Microsoft Purview compliance portal.
```

Comment:
Do you mean (the way is was presented was confusing.  This way is much clearer) -
 Removes the Microsoft Purview compliance portal or compliance cases from the Microsoft Purview compliance portal.